src/routes/v2/authenticatedSites/media.ts

```diff
@@ -108,22 +108,20 @@ export class MediaRouter {
     { directoryName: string },
     { directories: MediaDirOutput[] },
     never,
-    { page: number },
+    never,
     { userWithSiteSessionData: UserWithSiteSessionData }
   > = async (req, res) => {
     const { userWithSiteSessionData } = res.locals
-
     const { directoryName } = req.params
-    const { page } = req.query
 
     const {
       directories,
     } = await this.mediaDirectoryService.listMediaDirectoryContent(
       userWithSiteSessionData,
       {
         directoryName,
-        page,
-        limit: 0,
+        page: 0,
```

Comment:
hey am confused here, arent we supposed to use the page number from the fe, rather than defaulting to a value?

Reply:
this is to list the subdirectories, so we pass in sane defaults to avoid listing the actual pages/images

Reply:
we are not performing pagination here